tests/unit/ApprovalType.spec.ts

```diff
@@ -16,12 +16,16 @@ localVue.use(VueRouter)
 function createDefaultComponent (
   draftCourtOrderNumber: string = '',
   draftApprovedByRegistrar: boolean = false,
+  notice: string = '',
+  application: string = '',
   filingType: string = 'restoration'
 ): Wrapper<ApprovalType> {
   return mount(ApprovalType, {
     propsData: {
       draftCourtOrderNumber,
```

Comment:
I know they're the draft values the first time, but they're coming from the store, right? And you don't know if the store will change (and thus these props need to update things). So I recommend renaming these to simply "courtOrderNumber" and "approvedByRegistrar" (which would also line up nicely with the update events coming back from the component.